src/Sylius/Behat/Context/Ui/Shop/AccountContext.php

```diff
@@ -262,6 +262,14 @@ public function iBrowseMyOrders()
         $this->orderIndexPage->open();
     }
 
+    /**
+     * @When I try to open payment page
+     */
+    public function iTryToOpenPaymentPage()
+    {
+        $this->orderShowPage->pay();
```

Comment:
Is it paying or opening a payment page? For me, it's a little bit confusing what we try to do :/